### PR TITLE
Feature/elasticsearch encryption

### PR DESCRIPTION
**Pull request by vigneswararaomacharla**
This is the feature which identifies elastic search encryption is enabled or not.

### CODE REVIEW

File: hammer/library/aws/elasticsearch.py [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+        for domain_detail in domain_details:
+            is_logging = False
+            domain_encrypted = False
+            domain_name = domain_detail["DomainName"]
+            domain_id = domain_detail["DomainId"]
+            domain_arn = domain_detail["ARN"]
+            encryption_at_rest = domain_detail.get("EncryptionAtRestOptions")
+            node_to_node_encryption = domain_detail.get("NodeToNodeEncryptionOptions")
+            if encryption_at_rest and encryption_at_rest["Enabled"]:
+                domain_encrypted = True
```
Comment (oleksii-dowjones) [inline hammer/library/aws/elasticsearch.py]: Should we consider these as different options? If it's encrypted at rest and not in transit, report that it's not encrypted in transit. In the opposite case, report that it's not encrypted at rest.

File: hammer/reporting-remediation/reporting/create_elasticsearch_unencrypted_issue_tickets.py [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                        f"*Region*: {region}\n"
+                        f"*Domain ID*: {domain_name}\n"
+                    )
+
+                    issue_description += JiraOperations.build_tags_table(tags)
+
+                    issue_description += (
+                        f"*Recommendation*: Encrypt Elasticsearch domain. To enable encryption follow below steps: \n"
+                        f"1. Choose to create new domain. \n"
+                        f"2. Enable node-node encryption or encryption at rest options.\n"
```
Comment (oleksii-dowjones) [inline hammer/reporting-remediation/reporting/create_elasticsearch_unencrypted_issue_tickets.py]: Again, why should we enable either encryption at rest or encryption in transit and not both?

Comment (vigneswararaomacharla) [reply hammer/reporting-remediation/reporting/create_elasticsearch_unencrypted_issue_tickets.py]: Ok. Updated issue remediation description to encrypt for both (at rest and transit)

File: hammer/library/aws/elasticsearch.py (hunk 1 above)
Comment (vigneswararaomacharla) [reply hammer/library/aws/elasticsearch.py]: Make changes as per encryption type.

**Pull request merged**